Teams update (#163)
pretix/pretix-docs: docs/guides/teams.de.md

# Teams

Sie müssen nicht alle Events selbst managen.
pretix erlaubt Ihnen, Kolleg*innen einzuladen und dafür unterschiedliche Teams mit unterschiedlichen Berechtigungen anzulegen.

> Dieser Artikel erklärt, wie Sie die "Teams"-Einstellungen nutzen, Teams anlegen, Team-Einladungen verschicken, eine solche Einladung selbst annehmen und wie Sie die Zwei-Faktor-Authentifizierung einschalten.

Reviewer: Bitte Links hinzufügen wie im Originalartikel

## Voraussetzungen

> Um Teams anzulegen und die Berechtigungen zu verwalten, benötigen Sie selbst Administrator*innen-Rechte für das Veranstalterkonto.

Reviewer: Das hat sich mit dem Update ebenfalls geändert und wir müssen es entsprechend auch im Originaltext anpassen
```suggestion
Um Teams anzulegen und die Berechtigungen zu verwalten, benötigen Sie in Ihrem Veranstalterkonto für "Teams" die Berechtigung "Lesen und ändern".
```

> Wenn Sie keinen Zugriff auf ein Veranstalterkonto haben, suchen Sie wahrscheinlich den Abschnitt, in dem erklärt wird, wie Sie eine Team-Einladung annehmen (LINK).

Reviewer: Link hinzufügen

> ## How-To

Reviewer: ```suggestion
## Anleitung
```

> Navigieren Sie zu Ihrem :navpath:Veranstalterkonto →  :fa3-users: Teams:.

Reviewer: ```suggestion
Navigieren Sie zu :navpath:Ihr Veranstalter →  :fa3-users: Teams:.
```

Diese Seite zeigt Ihnen sämtliche Teams Ihres Veranstalterkontos.

> Wenn Sie an dieser Stelle in Ihrem Veranstalterkonto noch keine Änderungen vorgenommen haben, wird hier nur ein Team mit dem Namen "Administratoren" gelistet.

Reviewer: 1. Passiv vermeiden 
2. Bei mir heißt das auf Deutsch auch Administrators. Ist das immer so?
```suggestion
Wenn Sie an dieser Stelle noch keine Änderungen vorgenommen haben, enthält die Liste nur ein Team mit dem Namen "Administrators". 
```

Author: Ich habe bei neu erstellten Konten nachgesehen: Dort steht "Administratoren"

<img width="1418" height="497" alt="Image" src="https://github.com/user-attachments/assets/975cfd2a-404a-413c-842f-9727d0a9fc10" />

Reviewer: Okay, dann ist das ein Problem an meinem Ende und passt so.

Dieses Team hat sämtliche Berechtigungen und enthält nur Ihr Benutzer*innenkonto.

> Sie können die Mitglieder eines Teams und die API-Tokens sehen, wenn Sie den Button :btn-icon:fa3-list:: rechts neben dem Teamnamen anklicken.

Reviewer: Terminologie: immer "klicken", niemals "anklicken". 
"ansehen" ist dagegen die passendere Übersetzung für "view". 
```suggestion
Sie können die Mitglieder eines Teams und die API-Tokens ansehen, indem Sie den Button :btn-icon:fa3-list:: rechts neben dem Teamnamen klicken.
```

Reviewer: "einsehen" wie im nächsten Satz würde auch gut passen

> Die Berechtigungen und Einstellungen für ein Team können Sie einsehen und ändern, wenn Sie den Button :btn-icon:fa3-edit:: anklicken.

Reviewer: "indem" drückt die logische Beziehung "wie tu ich das" besser aus als "wenn". 
```suggestion
Die Berechtigungen und Einstellungen für ein Team können Sie einsehen und ändern, indem Sie den Button :btn-icon:fa3-edit:: anklicken.
```

## Ein neues Team anlegen

> Wenn Sie in Ihrem Veranstalterkonto zu Teams navigieren, klicken Sie den Button :btn-icon:fa3-plus: Create a new team:, um ein neues Team anzulegen.

Reviewer: Die Formulierung im Originaltext ist ja mal sowas von daneben. Ich weiß nicht was mich da geritten hat. 

```suggestion
Um ein neues Team anzulegen, navigieren Sie zu :navpath:Your organizer → :fa3-users: Teams: und klicken den Button :btn-icon:fa3-plus: Create a new team:. 
```

Author: Der Button heißt im Deutschen "Neues Team erstellen" – dann muss auch ein neues Icon für diesen Artikel erstellt werden?

Reviewer 2: Das Icon sollte das gleiche sein, oder?
Aber ist das kein deutscher Navigationspfad an der Stelle?

Reviewer: Das Icon ist das gleiche. Sorry, mein Verbesserungsvorschlag war höchstens halbgar. 

```suggestion
Um ein neues Team anzulegen, navigieren Sie zu :navpath:Ihr Veranstalter → :fa3-users: Teams: und klicken den Button :btn-icon:fa3-plus: Neues Team erstellen:. 
```

Die Option "Alle Veranstalter-Berechtigungen" bezieht sich auf sämtliche Einstellungen im Veranstalterkonto selbst, für die auf dieser Seite keine Optionen explizit aufgeführt sind.
Wenn die URL der Einstellungs-Seite Ihres Veranstalterkontos /control/organizer/ enthält, dann wird die Berechtigung für die Optionen auf dieser Seite über die vorher genannte Option vergeben.
Dies bezieht sich nur auf die Einstellungen, die nicht explizit auf dieser Seite unter "Veranstalter-Berechtigungen" auswählbar sind.

Im Abschnitt "Veranstaltungs-Berechtigungen" können Sie entweder Zugriff auf alle Veranstaltungs-Berechtigungen vergeben, indem Sie die Option "Alle Veranstaltungs-Berechtigungen" auswählen.
Wenn die URL der Einstellungs-Seite Ihrer Veranstaltung /control/event/ enthält, dann wird die Berechtigung für die Optionen auf dieser Seite über die vorher genannte Option vergeben.
Dies bezieht sich nur auf die Einstellungen, die nicht explizit auf dieser Seite unter "Veranstaltungs-Berechtigungen" auswählbar sind.

> Oder Sie wählen beispielsweise die folgenden individuellen Berechtigungen aus: "Allgemeine Einstellungen" auf "Lesen und ändern", "Produkte, Kontingente und Fragen" auf "Lesen und ändern", "Bestellungen" auf "Alle lesen" und "Gutscheine" auf "Lesen und ändern" setzen.

Reviewer: Bitte die Änderungsvorschläge zum Originaltext anwenden und erneut übersetzen

> Wenn Sie unterschiedliche Berechtigungen für andere Veranstaltungen vergeben möchten, müssen Sie ein neues Team anlegen und dafür die passenden Berechtigungen auswählen und dieses Team auf die gewünschten Veranstaltungen einschränken.

Reviewer: ```suggestion
Wenn Sie unterschiedliche Berechtigungen für andere Veranstaltungen vergeben möchten, dann legen Sie ein neues Team an. 
Wählen Sie die passenden Berechtigungen aus und schränken Sie das Team auf die gewünschten Veranstaltungen ein. 
```

> Die Namenskonventionen für Ihre Teamnamen und die Berechtigungen hängen von Ihren individuellen Anforderungen ab.

Reviewer: ```suggestion
Der Namen und die genauen Berechtigungen, die Sie vergeben, hängen von Ihren individuellen Anforderungen ab.
```

> Die Namen und Berechtigungen können auch später noch jederzeit verändert werden.

Reviewer: ```suggestion
Sie können sie auch später noch jederzeit verändern.
```

Author: Danke. Vielleicht noch besser?

```suggestion
Sie können diese auch später noch jederzeit verändern.
```

> Es zahlt sich aber aus und Sie vermeiden zukünftige Probleme und Verwirrungen, wenn Sie hier ein wenig Zeit investieren und Ihre Anforderungen und Berechtigungen vorher festlegen, bevor Sie Kolleg*innen in die jeweiligen Teams einladen.

Reviewer: ```suggestion
Sie können aber zukünftige Probleme und Verwirrung vermeiden, indem Sie Ihre Auswahl hier festlegen, bevor Sie Kolleg*innen in das Team einladen.
```

> Klicken Sie den Speichern Button, um das neu angelegte Team zu speichern.

Reviewer: ```suggestion
Klicken Sie den Button :btn:Speichern:, um das neu angelegte Team zu speichern.
```

> Danach werden Sie auf die Übersichtsseite der Team-Mitglieder weitergeleitet.

Reviewer: Passiv vermeiden
```suggestion
Das leitet Sie auf eine Seite mit einer Übersicht der Team-Mitglieder weiter. 
```

!!! Note
  Wenn ein Team-Mitglied keine Berechtigung für einen Bereich des Systems hat, kann es eventuell trotzdem Daten dieses Bereichs einsehen, wenn dieser mit anderen Bereichen in pretix verknüpft ist.
  Beispielsweise kann ein Team-Mitglied mit Berechtigung zum Zugriff auf die Bestellungen auch den Wertgutschein einsehen, der mit einer Bestellung angelegt oder zur Bezahlung einer Bestellung verwendet wurde.

## Eine Person ins Team einladen

Navigieren Sie zu :navpath:Veranstalterkonto → :fa3-users: Teams: und wählen Sie das Team aus, für das Sie eine Einladung verschicken möchten.

> Sie können eine Person einladen, indem Sie die E-Mail-Adresse der Person in das Eingabefeld unterhalb der Mitgliederliste des Teams eingeben oder einfügen und rechts neben dem Eingabefeld auf den Button "Hinzufügen" klicken.

Reviewer: Einfügen kann man immer auch. Wenn wir das jedes mal extra erwähnen, dann machen wir unsere Docs unnötig wortreich. 
Klicken statt "auf" klicken.
```suggestion
Sie können eine Person einladen, indem Sie die E-Mail-Adresse der Person in das Eingabefeld unterhalb der Mitgliederliste des Teams eingeben und rechts neben dem Eingabefeld den Button "Hinzufügen" klicken.
```

Wenn die eingeladene Person bereits über ein pretix-Benutzerkonto verfügt und dieser die gleiche E-Mail-Adresse verwendet, wird die Person sofort dem Team hinzugefügt.
In diesem Fall wird die Person nicht vom System benachrichtigt, weshalb Sie die Person darüber informieren sollten.

> Die Person wird entweder mit einem grauen oder grünen :fa3-shield: Symbol rechts neben der E-Mail-Adresse aufgelistet, abhängig davon, ob diese die Zwei-Faktor-Authentifizierung aktiviert haben oder nicht.

Reviewer: ```suggestion
Die Person wird entweder mit einem grauen oder grünen Schildsymbol :fa3-shield: rechts neben der E-Mail-Adresse aufgelistet, abhängig davon, ob diese die Zwei-Faktor-Authentifizierung aktiviert hat oder nicht.
```

Wenn die eingeladene Person noch kein pretix-Benutzerkonto hat, erhält sie eine E-Mail mit einer Einladung, einen Account anzulegen.
In diesem Fall erscheint rechts in der Mitglieder-Liste neben der E-Mail-Adresse ein :fa3-envelope-o: Symbol mit dem Status "eingeladen, Antwort ausstehend" und ein :btn-icon:fa3-rotate-right::-Button, um die Einladung erneut per E-Mail zu verschicken.

!!! Note
  Auf dieser Seite können Sie auch einen API-Token anlegen, der Zugriff auf die pretix API erlaubt. Der API-Token erhält die gleichen Berechtigungen, die für die Mitglieder des Teams gelten. Weitere Informationen zu API-Tokens finden Sie in unserer [API Dokumentation](https://docs.pretix.eu/en/latest/api/tokenauth.html).

## Eine Einladung annehmen

Wenn Sie zum Team eingeladen wurden und noch kein pretix-Benutzerkonto haben, erhalten Sie eine E-Mail wie oben auf dem Screenshot zu sehen.

> Wenn Sie den Link in der E-Mail anklicken, werden Sie zu pretix weitergeleitet und haben in dem Dialogfenster die Möglichkeit, ein Passwort anzugeben und den "Registrieren"-Button anzuklicken, um einen Account anzulegen.

Reviewer: ```suggestion
Wenn Sie den Link in der E-Mail anklicken, werden Sie zu einem Dialogfenster weitergeleitet, in dem Sie ein Passwort angeben und den Button :btn:Registrieren: klicken können, um ein Konto anzulegen.
```

Author: Dann hier auch klicken anstatt anklicken?
```suggestion
Wenn Sie den Link in der E-Mail klicken, werden Sie zu einem Dialogfenster weitergeleitet, in dem Sie ein Passwort angeben und den Button :btn:Registrieren: klicken können, um ein Konto anzulegen.
```

Reviewer 2: Hm, in Matzes Vorschlag stand doch schon "klicken" statt "anklicken"? Ist GitHub wieder seltsam?

Author: In dem ursprünglichen Satz stand anklicken/anzuklicken und Matze hat nur das anzuklicken geändert. Direkt am Anfang des Satzes steht noch ein "anklicken": "Wenn Sie den Link in der E-Mail **anklicken** [...]"

Reviewer: Völlig richtig. "öffnen" oder "dem Link folgen" wäre auch okay

> Sie werden daraufhin zum Dashboard weitergeleitet und die Nachricht "Willkommen zu pretix! Sie sind nun Mitglied vom Team "Mitarbeiter*innen".

Reviewer: ```suggestion
Sie werden daraufhin zum Dashboard weitergeleitet, das folgende Nachricht anzeigt: "Willkommen zu pretix! Sie sind nun Mitglied vom Team "Mitarbeiter*innen".
```

Damit haben Sie Zugriff auf das Veranstalterkonto und die Veranstaltungen, so wie die Berechtigungen im Team "Mitarbeiter*innen" es vorsehen.

Wenn Sie zu einem Team eingeladen wurden und bereits ein pretix-Benutzerkonto haben, können Sie sich in pretix einloggen und haben Zugriff auf das Veranstalterkonto und die Veranstaltungen, so wie die Berechtigungen im Team "Mitarbeiter*innen" es vorsehen.

> ## Zwei-Faktor-Authentifizierung einschalten

Reviewer: ```suggestion
## Zwei-Faktor-Authentifizierung voraussetzen
```
Alternativ 
```suggestion
## Zwei-Faktor-Authentifizierung erfordern
```

> Sie können alle Team-Mitglieder dazu verpflichten, die Zwei-Faktor-Authentifizierung zu verwenden.

Reviewer: ```suggestion
Sie können alle Mitglieder eines Teams dazu verpflichten, die Zwei-Faktor-Authentifizierung zu verwenden.
```

> Wählen Sie dazu ein bestehendes Team aus oder legen Sie ein neues an.

Reviewer: ```suggestion
Dazu bearbeiten Sie ein Team oder legen Sie ein neues an.
```

> Wählen Sie die Checkbox "Alle Mitglieder dieses Teams müssen Zwei-Faktor-Authentifizierung nutzen" an.

Reviewer: Auch im Original fehlt hier der Schritt mit dem Speichern-Knopf
```suggestion
Aktivieren Sie das Kontrollkästchen "Alle Mitglieder dieses Teams müssen Zwei-Faktor-Authentifizierung nutzen" und klicken Sie den Button :btn:Speichern:. 
```

> Es kann ein paar Minuten dauern, bis dieses Setting für alle Mitglieder des Teams gilt.

Reviewer: ```suggestion
Es kann ein paar Minuten dauern, bis diese Einstellung wirksam wird.
```

Wenn ein Team-Mitglied, das noch keine Zwei-Faktor-Authentifizierung verwendet, sich das nächste Mal in pretix einloggt, wird es auf die Seite "Zwei-Faktor-Authentifizierung" weitergeleitet.

> Es ist nicht möglich, diese Seite zu überspringen, bis die Zwei-Faktor-Authentifizierung aktiviert wurde.

Reviewer: Passiv vermeiden
```suggestion
Das Mitglied kann diese Seite erst verlassen, wenn es 2FA aktiviert (und ein Gerät hinzugefügt falls notwendig) oder das betreffende Team verlässt.
```

Author: Finde ich einen guten Vorschlag, aber mir fehlt das Hilfsverb "hat":
```suggestion
Das Mitglied kann diese Seite erst verlassen, wenn es 2FA aktiviert hat (und, falls notwendig, ein Gerät hinzugefügt hat) oder das betreffende Team verlässt.
```

> Wenn Sie mehr über die erhöhte Sicherheit erfahren möchten, welche die Zwei-Faktor-Authentifizierung mit sich bringt und wie Sie diese für Ihr Benutzerkonto einschalten, lesen Sie unseren Artikel zur Zwei-Faktor-Authentifizierung (LINK).

Reviewer: Link hinzufügen. Komma, da der eingeschobene Relativsatz dort aufhört
```suggestion
Wenn Sie mehr über die erhöhte Sicherheit erfahren möchten, welche die Zwei-Faktor-Authentifizierung mit sich bringt, und wie Sie diese für Ihr Benutzerkonto einschalten, lesen Sie unseren Artikel zur Zwei-Faktor-Authentifizierung (LINK).
```

> ## Zwei-Faktor-Authentifizierung einrichten

Reviewer: hmm, ja. Ist im Original auch nicht optimal gelöst. Vielleicht:
```suggestion
## Zwei-Faktor-Authentifizierung gemäß der Anforderung einrichten
```

> Wenn pretix Sie direkt nach dem Login zur Seite "Zwei-Faktor-Authentifizierung" weiterleitet, hat die Person, welche die Teams in Ihrem Veranstalterkonto verwaltet, die Option zur Verwendung der Zwei-Faktor-Authentifizierung verpflichtend für Ihr Benutzerkonto eingeschaltet.

Reviewer: Fehlendes Wort im Original
```suggestion
Wenn pretix Sie direkt nach dem Login zur Seite "Zwei-Faktor-Authentifizierung" weiterleitet und Sie keine andere Seite in pretix öffnen können, hat die Person, welche die Teams in Ihrem Veranstalterkonto verwaltet, die Option zur Verwendung der Zwei-Faktor-Authentifizierung verpflichtend für Ihr Benutzerkonto eingeschaltet.
```

> Um zu erfahren, wie Sie die Zwei-Faktor-Authentifizierung richtig einrichten, lesen Sie unseren Artikel zur Zwei-Faktor-Authentifizierung (LINK).

Reviewer: Link hinzufügen

Wenn Sie unterschiedliche Berechtigungen für andere Veranstaltungen vergeben möchten, dann legen Sie ein neues Team an. 
Wählen Sie die passenden Berechtigungen aus und schränken Sie das Team auf die gewünschten Veranstaltungen ein. 

Der Namen und die genauen Berechtigungen, die Sie vergeben, hängen von Ihren individuellen Anforderungen ab.
Die Namen und Berechtigungen können auch später noch jederzeit verändert werden.
Sie können aber zukünftige Probleme und Verwirrung vermeiden, indem Sie Ihre Auswahl hier festlegen, bevor Sie Kolleg*innen in das Team einladen.
Klicken Sie den Button :btn:Speichern:, um das neu angelegte Team zu speichern.
Das leitet Sie auf eine Seite mit einer Übersicht der Team-Mitglieder weiter. 

!!! Note
  Wenn ein Team-Mitglied keine Berechtigung für einen Bereich des Systems hat, kann es eventuell trotzdem Daten dieses Bereichs einsehen, wenn dieser mit anderen Bereichen in pretix verknüpft ist.
  Beispielsweise kann ein Team-Mitglied mit Berechtigung zum Zugriff auf die Bestellungen auch den Wertgutschein einsehen, der mit einer Bestellung angelegt oder zur Bezahlung einer Bestellung verwendet wurde.

## Eine Person ins Team einladen

> Navigieren Sie zu :navpath:Veranstalterkonto → :fa3-users: Teams: und wählen Sie das Team aus, für das Sie eine Einladung verschicken möchten.

Reviewer 2: ```suggestion
Navigieren Sie zu :navpath:Ihr Veranstalter → :fa3-users: Teams: und wählen Sie das Team aus, für das Sie eine Einladung verschicken möchten.
```

Sie können eine Person einladen, indem Sie die E-Mail-Adresse der Person in das Eingabefeld unterhalb der Mitgliederliste des Teams eingeben und rechts neben dem Eingabefeld den Button "Hinzufügen" klicken.
Wenn die eingeladene Person bereits über ein pretix-Benutzerkonto verfügt und dieser die gleiche E-Mail-Adresse verwendet, wird die Person sofort dem Team hinzugefügt.
In diesem Fall wird die Person nicht vom System benachrichtigt, weshalb Sie die Person darüber informieren sollten.
Die Person wird entweder mit einem grauen oder grünen Schildsymbol :fa3-shield: rechts neben der E-Mail-Adresse aufgelistet, abhängig davon, ob diese die Zwei-Faktor-Authentifizierung aktiviert hat oder nicht.

Wenn die eingeladene Person noch kein pretix-Benutzerkonto hat, erhält sie eine E-Mail mit einer Einladung, einen Account anzulegen.
In diesem Fall erscheint rechts in der Mitglieder-Liste neben der E-Mail-Adresse ein :fa3-envelope-o: Symbol mit dem Status "eingeladen, Antwort ausstehend" und ein :btn-icon:fa3-rotate-right::-Button, um die Einladung erneut per E-Mail zu verschicken.

!!! Note

>   Auf dieser Seite können Sie auch einen API-Token anlegen, der Zugriff auf die pretix API erlaubt. Der API-Token erhält die gleichen Berechtigungen, die für die Mitglieder des Teams gelten. Weitere Informationen zu API-Tokens finden Sie in unserer [API Dokumentation](https://docs.pretix.eu/en/latest/api/tokenauth.html).

Reviewer 2: ```suggestion
  Auf dieser Seite können Sie auch einen API-Token anlegen, der Zugriff auf die pretix-API erlaubt. 
  Der API-Token erhält die gleichen Berechtigungen, die für die Mitglieder des Teams gelten. 
  Weitere Informationen zu API-Tokens finden Sie in unserer [API-Dokumentation](https://docs.pretix.eu/en/latest/api/tokenauth.html).
```

Damit haben Sie Zugriff auf das Veranstalterkonto und die Veranstaltungen, so wie die Berechtigungen im Team "Mitarbeiter*innen" es vorsehen.

Wenn Sie zu einem Team eingeladen wurden und bereits ein pretix-Benutzerkonto haben, können Sie sich in pretix einloggen und haben Zugriff auf das Veranstalterkonto und die Veranstaltungen, so wie die Berechtigungen im Team "Mitarbeiter*innen" es vorsehen.

## Zwei-Faktor-Authentifizierung erfordern

Sie können alle Mitglieder eines Teams dazu verpflichten, die Zwei-Faktor-Authentifizierung zu verwenden.
Dazu bearbeiten Sie ein Team oder legen Sie ein neues an.
Aktivieren Sie das Kontrollkästchen "Alle Mitglieder dieses Teams müssen Zwei-Faktor-Authentifizierung nutzen" und klicken Sie den Button :btn:Speichern:. 
Es kann ein paar Minuten dauern, bis diese Einstellung wirksam wird.

Wenn ein Team-Mitglied, das noch keine Zwei-Faktor-Authentifizierung verwendet, sich das nächste Mal in pretix einloggt, wird es auf die Seite "Zwei-Faktor-Authentifizierung" weitergeleitet.
Es ist nicht möglich, diese Seite zu überspringen, bis die Zwei-Faktor-Authentifizierung aktiviert wurde.

> Wenn Sie mehr über die erhöhte Sicherheit erfahren möchten, welche die Zwei-Faktor-Authentifizierung mit sich bringt, und wie Sie diese für Ihr Benutzerkonto einschalten, lesen Sie unseren Artikel zur Zwei-Faktor-Authentifizierung (LINK).

Reviewer 2: > Wenn Sie mehr über die erhöhte Sicherheit erfahren möchten, welche die Zwei-Faktor-Authentifizierung mit sich bringt, und wie Sie diese für Ihr Benutzerkonto einschalten, lesen Sie unseren Artikel zur Zwei-Faktor-Authentifizierung (LINK).

Bitte den Link einfügen
